src/picklescan/torch.py

```diff
@@ -61,7 +61,7 @@ def _is_zipfile(f) -> bool:
     start = f.tell()
 
     byte = f.read(1)
-    while byte != "":
+    while byte != b"":
```

Comment:
Thanks for porting the fix! Can you also update the link line 50 so it remains in sync? Something like https://github.com/pytorch/pytorch/blob/0b3316ad2c6ff61416597ef29e8865876dcb12f5/torch/serialization.py#L66 should do.